web/index.templ

```diff
@@ -52,6 +52,9 @@ templ base(title string, body templ.Component) {
     }
       </style>
 			@templ.JSONScript("anubis_version", anubis.Version)
+			for key, value := range ogTags {
+            		<meta property={ key } content={ value }/>
+            	}
```

Comment:
unrelated: that file should be :retabbed 😅 mixing spaces and tabs in one line is usually up for trouble.

Reply:
O true, suprised gofmt didn't fix that- maybe I didn't run it